src/main/java/mate/academy/bookshopapp/controller/BookController.java

```diff
@@ -38,14 +38,14 @@ public BookDto updateBookById(@PathVariable Long id, @RequestBody BookDto bookDt
         return bookService.updateBookById(id, bookDto);
     }
```

Comment:
Use CreateBookRequestDto or create separate UpdateBookRequestDto and use Valid annotation ;)